MediaWiki - Update test manifest with mandatory properties definition

sunilnatraj opened the pull request:
Support for definition of mandatory properties and enforcement for new media upload

sunilnatraj committed:
Issue 95

Support for definition of mandatory properties and enforcement for new media upload

sunilnatraj committed:
Updated manifest & schema

Added the key for mandatory properties for new media upload

sunilnatraj commented:
@wetneb Can you review this

wetneb commented on biblissima-data-manifest-v1.json:
```diff
@@ -4,7 +4,8 @@
         "main_page": "https://data.biblissima.fr/w/Accueil",
-        "api": "https://data.biblissima.fr/w/api.php"
+        "api": "https://data.biblissima.fr/w/api.php",
+        "mandatoryMediaInfoPropertyIds": "P7482, P170, P6216"
```
I don't think we should provide `mandatoryMediaINfoPropertyIds` if the Wikibase instance doesn't use MediaInfo entities. Especially since the property identifiers you added here probably don't make sense on that Wikibase instance.
So I think I would only add it to the Wikimedia Commons manifest. It should also use an array syntax.

sunilnatraj committed:
Feedback addressed

Adding the key for mandatory properties for new media upload only in test & wikimedia manifest
